src/app/screens/Home/index.tsx

```diff
@@ -242,19 +244,21 @@ function Home() {
               onClick={async () => {
                 try {
                   setLoadingSendSats(true);
-                  const origin = {
+                  const originData = {
                     external: true,
                     name: lnData[0].name,
                     host: lnData[0].host,
                     description: lnData[0].description,
                     icon: lnData[0].icon,
                   };
                   if (lnData[0].method === "lnurl") {
-                    navigate(
-                      `/lnurlPay?lnurl=${
-                        lnData[0].address
-                      }&origin=${encodeURIComponent(JSON.stringify(origin))}`
-                    );
+                    // HOW CAN THIS BE TESTED ON HOME !?!
```

Comment:
go to some website, or twiter or some youtube video that has the lightning address in it and click on the Alby icon. 

https://twitter.com/escapedcat
https://www.youtube.com/watch?v=RFSBWrAllzw

Reply:
This still works 👍